cbrain_task/civet_combiner/views/_task_params.html.erb

```diff
@@ -22,26 +22,29 @@
 #
 -%>
 
-<strong>Name of the combined Study:</strong><BR>
+<strong>Name of the combined Study:</strong>
+<br>
 <%= text_field_tag :civet_study_name.to_la, params[:civet_study_name] %>
 <br>
 
 <table>
   <tr>
     <th>CivetOutput Names To Be Merged</th>
   </tr>
-<% params[:interface_userfile_ids].each do |id| %>
-  <tr>
-    <% u = Userfile.find_by_id(id) %>
-    <% if u.is_a?(CivetOutput) || u.is_a?(CivetStudy) %>
-      <td>OK: <%= h(u.class.to_s) %> <%= link_to_userfile_if_accessible u %> <%= hidden_field_tag 'cbrain_task[params][civet_collection_ids][]', id %></td>
-    <% else %>
-      <td>Error: <%= link_to_userfile_if_accessible u %> is NOT a CivetOutput or CivetStudy.</td>
-    <% end %>
-  </tr>
-<% end %>
+  <% params[:interface_userfile_ids].each do |id| %>
+    <tr>
+      <% u = Userfile.find_by_id(id) %>
+      <% if u.is_a?(CivetOutput) || u.is_a?(CivetStudy) %>
+        <td>OK: <%= h(u.class.to_s) %> <%= link_to_userfile_if_accessible u %> <%= hidden_field_tag 'cbrain_task[params][civet_collection_ids][]', id %></td>
+      <% else %>
+        <td>Error: <%= link_to_userfile_if_accessible u %> is NOT a CivetOutput or CivetStudy.</td>
+      <% end %>
+    </tr>
+  <% end %>
 </table>
 
 After the Study is successfully created, <strong>destroy permanently</strong>
-the CivetOutputs used? <%= check_box_tag( :destroy_sources.to_la, 'YeS') %>
+the CivetOutputs used? <%= check_box_tag( :destroy_sources.to_la, 'Yes') %>
```

Comment:
The form shoudl ask the copy/no copy quesiton first, and if the copy is set to true, then offer the question for destroy the source. It doesnt' have to be dynamic just something like:

```
Copy civet outputs yes/no (not recomended)
If copying, destroy civet outputs after copy? yes.no
```